packages/pharos/src/components/alert/pharos-alert.scss

```diff
@@ -13,11 +13,12 @@
   padding: var(--pharos-spacing-1-x);
   border-radius: var(--pharos-radius-base-standard);
   display: grid;
-  grid-template-areas: 'icon body';
-  grid-template-columns: var(--pharos-spacing-one-and-a-half-x) 1fr;
+  grid-template-areas: 'icon body dismiss';
+  grid-template-columns: var(--pharos-spacing-one-and-a-half-x) 1fr 1.75rem;
```

Comment:
Should this _only_ be done when the alert is closable?

Reply:
I guess so, or else there is the space allotted for it whether it is closable or not. Though, visually it looks the same.

Reply:
I think for alerts in smaller screen real estate they now span across 1–2 additional lines of text and that text looks like it's a bit off to the left compared to before. It seems good to give them the real estate if possible, considering that most alerts at the moment are _not_ closable.